packages/api/src/routes/webhook/fitbit.ts

```diff
@@ -0,0 +1,51 @@
+import { Request, Response } from "express";
+import Router from "express-promise-router";
+import status from "http-status";
+import { asyncHandler } from "../util";
+import { processData } from "../../command/webhook/fitbit";
+import { getEnvVar } from "../../shared/config";
+
+const routes = Router();
+
+/** ---------------------------------------------------------------------------
+ * GET /webhook/fitbit
+ *
+ * Verify Fitbit WH subscriber. Receives a verification notification and sends back a verification post request.
+ *
+ */
```

Comment:
as per my previous comment, should we do this in a lambda so someone can't spam our backend?

Reply:
Also b/c this endpoints under `/webhook` are not accessible from the internet.